js/components/buttons/follow.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+<button type="button" class="btn btn-follow" :class="btnClasses" @click="toggle"
+    v-tooltip="tooltip" :tooltip-placement="tooltipPlacement">
+    <span class="fa" :class="icon"></span>
+    <span v-if="withLabel">{{ label }}</span>
+    <span v-if="followers">{{ followers }}</span>
+</button>
+</template>
+
+<script>
+import i18n from 'i18n';
+import Auth from 'auth';
+
+export default {
+    props: {
+        classes: {
+            type: Array,
+            coerce(value) {
+                if (Array.isArray(value)) return value;
+                return value.split(' ').filter(value => value.trim());
+            }
+        },
+        followers: {
+            type: Number,
+            default: undefined
+        },
+        following: {
+            type: Boolean,
+            default: false
+        },
+        withLabel: {
+            type: Boolean,
+            default: false
+        },
+        tooltip: {
+            type: String,
+            default: i18n._("I'll be informed about its activity")
+        },
+        tooltipPlacement: {
+            type: String,
+            default: 'left'
+        },
+        url: {
+            type: String,
+            required: true
+        }
+    },
+    computed: {
+        btnClasses() {
+            const classes = {active: this.following};
+            this.classes.forEach(cls => classes[cls] = true);
```

Comment:
We hit [no-return-assign](http://eslint.org/docs/rules/no-return-assign) with @yohanboniface the other day and given that arrow function implicitly return the statement, StandardJS consider that you must add mustaches in these ambiguous cases. Thoughts?